.github/workflows/canary-integration-test.yml

```diff
@@ -39,6 +39,22 @@ jobs:
         with:
           use-tmate: ${{ secrets.USE_TMATE }}
 
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          # this might remove tools that are actually needed,
+          # if set to "true" but frees about 6 GB
+          tool-cache: true
+
+          # all of these default to true, but feel free to set to
```

Comment:
If these all default to true, how about we remove them? Since we copy this action so many places, no need to copy the defaults everywhere.

Reply:
removed